docs/rules/no-test-only.md

```diff
@@ -0,0 +1,26 @@
+# uvu/no-test-only
+
+Warn about forgotten `test.only()`.
+
+## Rationale
+
+It is popular mistake to forget `test.only()` from debug session.
+
+## Examples
+
+Examples of **incorrect** code for this rule:
+
+```javascript
+test.only();
```

Comment:
Let's add a full example with test description and test body

Reply:
Added full test example.

Reply:
I see old one in `File changed`. Maybe you forgot to `git push`?

Reply:
Yep. 
Now I pushed changes.